app/views/help/index.html.erb

```diff
@@ -4,60 +4,53 @@
         content="Find out about GOV.UK, including the use of cookies, accessibility of the site, the privacy policy and terms and conditions of use." />
 <% end %>
 
-<main id="content" role="main" class="group help-page">
-  <header class="page-header group">
-    <div class="inner">
+<main id="content" role="main">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
       <%= render "govuk_publishing_components/components/title", title: "Help using GOV.UK" %>
+      <p class="govuk-body">Find out about GOV.UK, including the use of cookies, accessibility of the site, the privacy policy and terms and conditions of use.</p>
+      <ul class="govuk-list">
+        <li>
+          <a href="/help/about-govuk" class="govuk-link govuk-!-font-size-24 govuk-!-font-weight-bold">About GOV.UK</a>
```

Comment:
We're using a margin and font-weight override on every link here, which doesn't seem quite right?

Reply:
I was trying to avoid writing more custom CSS for this application, but yeah, it is kind of ugly isn't it.

Reply:
i did the same in https://github.com/alphagov/feedback/pull/803, avoiding custom css since we have overrides avaialble